### PR TITLE
[KOGITO-2799] Harmonize Jenkinfiles

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-2799

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -68,15 +67,14 @@ pipeline{
...
-                    IMAGES.each{ image -> build_stages["${image}"] = {
-                            createWorkspace("$image")
-                            copyWorkspace("$image")
-                            dir(getWorkspacePath("$image")){
+                    IMAGES.each{ image -> build_stages[image] = {
+                            initWorkspace(image)
+                            dir(getWorkspacePath(image)){
                                 try{
-                                    sh "make ${image}"
+                                    sh "make ${image} ignore_test=${params.SKIP_TESTS} cekit_option='--work-dir .'"
```
Comment (Kaitou786) [inline Jenkinsfile]: On our pr's why would we want to skip the tests?

Comment (Kaitou786) [reply Jenkinsfile]: And where are you passing the param?

Comment (radtriste) [reply Jenkinsfile]: Forgot to remove it ... will do. good catch !

**Review by Kaitou786 — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

**Comment by radtriste** [comment]
@ricardozanini Can we merge that one ?
This should bring more stability to PR checks as each build&test cekit command is using it own cache, so no parallel conflict which may occur sometimes ...

**Pull request merged**